internal/admission/limitrange.go

```diff
@@ -0,0 +1,11 @@
+package admission
+
+import (
+	"github.com/kanopy-platform/hedgetrimmer/internal/limitrange"
+	corev1Listers "k8s.io/client-go/listers/core/v1"
+)
+
+type LimitRanger interface {
+	corev1Listers.LimitRangeLister
+	NewConfig(namespace string) (*limitrange.Config, error)
```

Comment:
A constructor style naming seems odd in an interface.

```suggestion
	LimitRangeConfig(namespace string) (*limitrange.Config, error)
```